src/Model/GroundWaterFlow/gwf3npf8.f90

```diff
@@ -1754,6 +1754,15 @@ subroutine read_grid_data(this)
     ! -- set ik33 flag
     if (lname(3)) then
       this%ik33 = 1
+    else if (.not. lname(3) .and. this%intvk > 0) then
+      this%ik33 = 1
+      if (this%ik33overk /= 0) then
+        write (errmsg, '(a)') 'K33OVERK option specified but K33 not specified.'
+        call store_error(errmsg)
+      end if
```

Comment:
@emorway-usgs, I see that this error message has been duplicated.  Is there any way to rework this logic so that the error message doesn't need to be duplicated?

Reply:
Sorry about that, attempted to condense logic with 8301c7c